### DISCUSSION TITLE
Option to step through changes

### DISCUSSION THREAD

**Discussion by thumDer** [outline/outline#5693] [Ideas]
It would be great to have a control on the revision views which allows the user to step through each and every changes that are part of the current revision. Currently the user has to go through the whole document and look for the highlighted parts, which is quite cumbersome on long documents. The way it is implemented in vs vode is quite convinient:
![kép](https://github.com/outline/outline/assets/54556708/09393bcc-35fd-45e4-a5ca-efe84c4049ac)

**Comment by tommoor** [comment]
Converted to an [issue](https://github.com/outline/outline/issues/5694), seems like a no-brainer.
